README.md

```diff
@@ -1,2 +1,31 @@
-# AgPypeline
-Python library of common classes and functions
+# Transformer: agpypeline
+<img src="https://github.com/az-digitalag/Drone-Processing-Pipeline/raw/07b1edc34a1faea501c80f583beb07f9d6b290bb/resources/drone-pipeline.png" width="100" />
+
+This is an installable package derived from the [base-image](https://github.com/AgPipeline/base-docker-support/tree/master/base-image) and the
+[base-transformer-class](https://github.com/AgPipeline/drone-pipeline-environment/tree/master/base-transformer-class)
+
+## Expected metadata
+There are several metadata key/value pairs that this transformer expects to receive.
+The metadata keys listed below are all defined in the [BRAPI V1.3](https://brapi.docs.apiary.io/#) standard.
+
+*  studyName - the name of the study the data belongs to
+*  season - the season associated with the data
+*  observationTimeStamp - a timestamp override in ISO 8610 long format
+*  germplasmName - the name of the crop being tested in the plot
+*  collectingSite - site identification
+
+If the observationTimeStamp metadata key is not specified, the EXIF information in source image files are checked and the earliest found timestamp will be used. 
+
```

Comment:
Add a sentence about how the current timestamp is used if no timestamp is found.